### PR TITLE
Milestone 3: Creating portfolio form

### PR DESCRIPTION
**Pull request by Dangelo-JAN**
## Portfolio's Form milestone 3: Creating portfolio form

In this milestone we:

- Update index.thml
- Update style.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -177,4 +177,62 @@ <h4 class="g-item10">About Me</h4>
...
+          class="email-field"
+          type="email" 
+          id="mail" 
+          name="user_email"
+          placeholder="Dangelo@email.com"
+          required
+        >
+      </li>
+      <li class="li-form">
+        <textarea
```
Comment (codepantha) [inline index.html:213]: - [ ] Let's make sure to set a maximum length of allowed characters to 500 for this textarea as per requirements 🤓

File: index.html [hunk 2]
```diff
@@ -177,4 +177,62 @@ <h4 class="g-item10">About Me</h4>
...
+          class="msg-field"
+          id="msg" 
+          name="user_message"
+          placeholder="Enter text here"
+          required
+        ></textarea>
+      </li>
+      </ul>
+      <div class="centering-s">
+        <button class="g-item8 mr-s" type="submit">See Project</button>
```
Comment (codepantha) [inline index.html:224]: - [ ] Please implement the button hover and active state as shown in the Figma template 🙏

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @Dangelo-JAN 👋, 

You have done a great job so far 👏 👏 
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes ♻️ 
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 3]
```diff
@@ -177,4 +177,63 @@ <h4 class="g-item10">About Me</h4>
...
+  <!--Form section-->
+  <section class="c-container" id="form-section">
+    <h5 class="h-form">CONTACT US</h5>
+    <P class="p-form">
+      If you have an application you are interested in developing, 
+      a feature that you need built or a project that needs coding.
+       I’d love to help with it 
+    </P>
+    <form action="#form-section" method="post">
+      <ul class="non-style">
```
Comment (keroloussamy) [inline index.html]: I think here you made a small mistake, you have changed the `action` with id class name. Kindly put the link as you did before.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @Dangelo-JAN, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

 ## Required Changes ♻️
_Check the comments under the review._





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — APPROVED** [review]
Hi @Dangelo-JAN, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**